output-comparator/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation 'com.beust:jcommander:1.48'
     implementation 'com.google.code.gson:gson:2.8.6'
     implementation 'com.github.stefanbirkner:system-lambda:1.2.0'
+    implementation 'uk.org.webcompere:system-stubs-core:2.0.0'
```

Comment:
Sorry for coming in late here, but if this is just used in tests then I believe this should be `testImplementation` similar to the below line. Otherwise we're going to be packaging this dependency with the release JAR.

Reply:
Looks like latest version is `2.0.1`

Reply:
> Sorry for coming in late here, but if this is just used in tests then I believe this should be `testImplementation` similar to the below line. Otherwise we're going to be packaging this dependency with the release JAR.

I was not aware of this distinction. I will know for next time :) Everything is fixed in #1140